packages/react/react/src/hooks/useDebounce.ts

```diff
@@ -1,4 +1,4 @@
-import debounce from 'lodash/debounce';
+import { debounce, type DebounceSettings } from 'lodash';
```

Comment:
Please use only `lodash/debounce` for bundling optimization.
See [per-method-packages](https://lodash.com/per-method-packages).

Reply:
You can use `Parameters` to extract the type of options.
https://github.com/toss/slash/blob/9b7ee8a03fc719a56858b7c16e4ff37a11b1cc0e/packages/react/react/src/hooks/useThrottle.ts#L18

Reply:
thank you for comment about bundling optimization!
and I used Utility type! 💪

Reply:
[fix: @toss/react useDebounce build optimization](https://github.com/toss/slash/pull/95/commits/13e91dd2da9b0cc91ee48ff92659ec0a2393081b)
[13e91dd](https://github.com/toss/slash/pull/95/commits/13e91dd2da9b0cc91ee48ff92659ec0a2393081b)